components/Header/Header.js

```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components";
+
+const HeaderContainer = styled.div`
```

Comment:
using the header element would be more semantic:

```suggestion
const HeaderContainer = styled.header`
```